src/components/GenreList.jsx

```diff
@@ -1,20 +1,38 @@
 import '../styles/GenreList.scss';
 import PropTypes from 'prop-types';
+import { useSearchParams } from 'react-router-dom';
 
-const GenreBtn = ({genre, isSelected, onGenreSelection}) => {
+const GenreBtn = ({genre, selectedGenre, onGenreSelection}) => {
     return (
         <li>
-            <button className={"genre-item " + (isSelected ? "selected" : "")} onClick={() => onGenreSelection(genre)} >{genre} {isSelected}</button>
+            <button className={"genre-item " + (selectedGenre === genre ? "selected" : "")} onClick={() => onGenreSelection(genre)} >{genre}</button>
         </li>
     )
 }
 
-const GenreList = ({ genresList, selectedGenre, handleSelectGenre }) => {
+const GenreList = ({ genresList }) => {
```

Comment:
add type for genreList

Reply:
Module 10 has task to move all code to typescript